src/pages/dev/axelarjs-sdk/tx-status-query-recovery.mdx

```diff
@@ -65,12 +94,9 @@ enum GasPaidStatus {
 }
 ```
 
-### Trigger manual relay of transaction through the Axelar network
+## Manually relay the transaction through the Axelar network
 
-The following method, once invoked, will:
-
-1. Query the current status of the transaction to be in one of the states above.
-2. Recover from source to destination if needed.
+Use [`manualRelayToDestChain()`](https://github.com/axelarnetwork/axelarjs-sdk/blob/86bc55224f84f9539dd109e5dfd6b443a116eebe/src/libs/TransactionRecoveryApi/AxelarGMPRecoveryAPI.ts#L453) to manually relay a transaction to the destination chain through the Axelar network. It will query the current transaction status and recover from source to destination if needed.
```

Comment:
maybe just mention that this function is designed to be used before the gateway approve step, in other words a tx stuck at this step
https://testnet.axelarscan.io/gmp/0x111ff754b85538afcfea5aa59d01f976eb9a7b7b45b483c3fb2c48833bf0780d-4